src/planscape/config/conditions.json

```diff
@@ -565,7 +593,8 @@
                   "data_year": "2021",
                   "reference_link": "https://data.fs.usda.gov/geodata/rastergateway/regional-resource-kit/docs/RRK_Metric_Dictionary_v2.pdf#page=34",
                   "min_value": 0,
-                  "max_value": 1000
+                  "max_value": 1000,
```

Comment:
does not make sense

Reply:
Is there a way to easily verify whether the max value was incorrectly input?